tests/gce_virtual_machine_test.py

```diff
@@ -51,14 +51,14 @@ def testInvalidFloat(self):
     with self.assertRaises(errors.Config.InvalidValue) as cm:
       self.decoder.Decode('1280.9.8MiB', _COMPONENT, _FLAGS)
     self.assertEqual(str(cm.exception), (
-        'Invalid test_component.memory value: "1280.9.8MiB". "1280.9.8" is not '
-        'a valid float.'))
+        'Invalid test_component.memory value: "1280.9.8MiB". Examples of valid '
+        'values: "1280MiB", "7.5GiB".'))
 
   def testNonIntegerMiB(self):
     with self.assertRaises(errors.Config.InvalidValue) as cm:
-      self.decoder.Decode('7.6GiB', _COMPONENT, _FLAGS)
+      self.decoder.Decode('7.5MiB', _COMPONENT, _FLAGS)
```

Comment:
Why was this changed? 7.6GiB is not an integer number of MiB, so as far as I can tell, gcloud does not support it (https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type#create). This test case was a good contrast to the 7.5GiB one in testValidStrings, and we should verify it's still rejected.